### PR TITLE
Added Device Support HMIP-DRDI3

### PR DESCRIPTION
**Pull request by mayrd**
- Added Channels, DeviceTypes in base
- Added support in hmip_cli.py (support of channelIndex for dimming and allow switch state)
- Minor edits to .gitignore and setup.py

### CODE REVIEW

File: homematicip/device.py [hunk 1]
```diff
@@ -1410,6 +1410,21 @@ class WiredDimmer3(Dimmer):
...
+class DinRailDimmer3(Dimmer):
+    """ HMIP-DRDI3 (Dimming Actuator Inbound 230V – 3x channels, 200W per channel) electrical DIN rail """
+
+    # copied from Switch, as they can also be switched
+    def set_switch_state(self, on=True, channelIndex=1):
+        data = {"channelIndex": channelIndex, "deviceId": self.id, "on": on}
+        return self._restCall("device/control/setSwitchState", body=json.dumps(data))
+
+    def turn_on(self, channelIndex=1):
+        return self.set_switch_state(True, channelIndex)
```
Comment (coreGreenberet) [inline homematicip/device.py:1422]: Could you please add this to the test method?

File: homematicip/device.py [hunk 2]
```diff
@@ -1410,6 +1410,21 @@ class WiredDimmer3(Dimmer):
...
+
+    def turn_off(self, channelIndex=1):
+        return self.set_switch_state(False, channelIndex)
```
Comment (coreGreenberet) [inline homematicip/device.py:1425]: Could you please add this to the test method?

File: homematicip/device.py (hunk 1 above)
Comment (mayrd) [reply homematicip/device.py]: Removed again and based on Dimmer class

File: homematicip/device.py [hunk 3]
```diff
@@ -1423,7 +1423,7 @@ def from_json(self, js):
         super().from_json(js)
         channels = get_functional_channels("MULTI_MODE_INPUT_DIMMER_CHANNEL", js)
         if channels:
-            self.c1dimLevel = channels[1]["dimLevel"]
+            self.c1dimLevel = channels[0]["dimLevel"]
```
Comment (coreGreenberet) [inline homematicip/device.py:1426]: you could use self.set_attr_from_dict, that way you won't even need to implement __str__

**Pull request merged**